src/library/Avatar/__tests__/__snapshots__/Avatar.spec.js.snap

```diff
@@ -1,192 +1,171 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Avatar demo examples Snapshots: basic 1`] = `
-.css-0 > *,
-[data-css-0] > * {
+.emotion-4 > * {
   margin-right: 1rem;
   margin-bottom: 1rem;
 }
 
-.css-1,
-[data-css-1] {
+.emotion-0 {
   box-sizing: border-box;
   color: #ffffff;
-  font-family: "Open Sans", -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-family: "Open Sans",-apple-system,system-ui,BlinkMacSystemFont,"Segoe UI",Roboto,Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
   font-size: 16px;
   line-height: 2.5em;
   outline: 0;
   -moz-osx-font-smoothing: auto;
   -webkit-font-smoothing: antialiased;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
   align-items: center;
```

Comment:
It makes me inordinately happy that vendor prefixed properties are now grouped with the non-vendor-prefixed one and, more importantly, are printed first so that the non-vendor-prefixed one wins if supported. Thanks, emotion! 👩‍🎤